addons/website/static/src/snippets/s_dynamic_snippet_carousel/000.scss

```diff
@@ -1,13 +1,16 @@
 .s_dynamic {
   .carousel-control-prev, .carousel-control-next {
     position: absolute;
-    text-shadow: 0px 0px 3px #343A40;
     width: 4rem;
-  }
-  .carousel-control-prev{
-    left: -4rem
-  }
-  .carousel-control-next{
-    right: -4rem
+    pointer-events: none;
+
+    > a {
+      display: block;
+      width: 1.5rem;
+      height: 1.5rem;
+      border-radius: 50%;
+      background-color: #343A40;
+      pointer-events: auto;
+    }
```

Comment:
- Mmh, not sure about making only the arrow clickable and not the full height area like in normal carousel. I understand the reason because somehow the whole card is in fact clickable too... but this only worsen one problem: when you switch slides, sometimes the arrow jumps because the cards are smaller in the next slide and since now the clickable area is small, if the user does click, click, click to switch multiple slides without moving its mouse, it can end up clicking on the card instead of the arrow :/ Either we let that next-slide clickable arrow be full height or we need to fix so that the height stays the same... or should we not care for stable versions? What do you think @bvr-odoo @JKE-be ? Other reason would be to not have to change the XML which custo may rely on, even if it is static. I think I would just let the clickable area be full height to minimize the stable diff and we'll do better in master.

- Do not hardcode the black color, ideally use a bootstrap carousel-related variable or at worst a bootstrap gray like `gray('700')`

- The arrows appears not centred to me, especially on Firefox. I think it requires hackish solutions to make it appear centred but I think it is worth it for the user eyes :wink: Maybe see with @bvr-odoo if needed :+1:

Reply:
![FireShot Capture 390 - Amazon fr _ livres, DVD, jeux vidéo, musique, high-tech, informatique_ - www amazon fr](https://user-images.githubusercontent.com/52911687/103890229-fa2f3f80-50e7-11eb-86d6-973fdb714401.jpg)
We could do as here (Amazon) -> increase the height of the clickable area without taking the full height.. what do you think ?

Reply:
About the arrows, you could use 'fa-chevron-circle-right' instead of 'fa-chevron-right' to get something pefectly centered

Reply:
If @qsm-odoo is agree :)
For centered arrows, try to use `chevron-circle-right` instead of `chevron-right` if you find a way to do it without change the xml. Maybe, using `:before` like in the options of the carousel snippets ... (grep `.s_carousel_bordered ` class)
If not possible, try the following code and apply the css on the <span> instead of the <a>

```
display: block;
width: 2rem;
height: 2rem;
border-radius: 50%;
background-color: #343A40;
pointer-events: auto;
line-height: 2rem;
padding-left: 0.2rem; <------ for the right arrow
padding-right: 0.2rem; <------ for the left arrow
```

Reply:
It's actually the best idea but you can do that change in xml, if it is just about one class and not changing the whole structure it is fine :+1:

Reply:
![circle](https://user-images.githubusercontent.com/8981640/103904912-d4a93280-50f5-11eb-9b03-1ad7f71376fc.png)
@bvr-odoo Is this what you meant?

Reply:
not really, If you use `chevron-circle-right` you dont need to set a background. I proposed 2 solutions and you took both instead of choosing one ;)

Reply:
also dont remove `fa-2x` otherwise it's too small to be seen

Reply:
You're right :)
![circle](https://user-images.githubusercontent.com/8981640/103906374-bc3a1780-50f7-11eb-9a5a-3944bcc939ec.png)